### PR TITLE
delete profile

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
https://user-images.githubusercontent.com/23293248/200788154-92f60764-37d7-4a88-8dae-f2e7da8fd9fc.MP4

# Test Plan
- Try deleting account from setting screen. Make sure you don't test with your main account 😆

### CODE REVIEW

File: packages/app/components/settings/settings-account-slot.tsx [hunk 1]
```diff
@@ -56,15 +65,55 @@ export const SettingAccountSlotFooter = () => {
...
+              Are you sure you want to delete your account? This action cannot
+              be undone
+            </Text>
+            <Text tw="py-4 text-base text-gray-900 dark:text-gray-100">
+              Type your <Text tw="font-bold">username</Text> to confirm.
+            </Text>
+          </View>
+          <Fieldset
+            onChangeText={(t) => (username.current = t)}
+            placeholder="@username"
```
Comment (alantoa) [inline packages/app/components/settings/settings-account-slot.tsx]: if the placeholder is real `username` will be better, just like GitHub remove the repo, wdyt?

**Review by alantoa — APPROVED** [review]
LGTM 🔥

File: packages/app/components/settings/settings-account-slot.tsx (hunk 1 above)
Comment (intergalacticspacehighway) [reply packages/app/components/settings/settings-account-slot.tsx]: i kept it like that to make it harder. But we can do it the Github way too

Comment (alantoa) [reply packages/app/components/settings/settings-account-slot.tsx]: 🤣 that makes sense!

Comment (intergalacticspacehighway) [reply packages/app/components/settings/settings-account-slot.tsx]: updated

**Pull request merged**